locales/en/plugin__kubevirt-plugin.json

```diff
@@ -532,12 +532,12 @@
   "Guest agent is required": "Guest agent is required",
   "Guest login credentials": "Guest login credentials",
   "Guest management": "Guest management",
-  "Guest system access log": "Guest system access log",
-  "Guest system access logs are disabled at cluster": "Guest system access logs are disabled at cluster",
-  "Guest system access logs are disabled at VirtualMachine": "Guest system access logs are disabled at VirtualMachine",
-  "Guest system access logs disabled at cluster": "Guest system access logs disabled at cluster",
-  "Guest system access logs not ready. a restart required": "Guest system access logs not ready. a restart required",
+  "Guest system log": "Guest system log",
   "Guest system log access": "Guest system log access",
+  "Guest system logs are disabled at cluster": "Guest system logs are disabled at cluster",
```

Comment:
Just noting that there are still two cluster-related lines (537 and 539) if you meant to resolve that in this PR.

Reply:
I'd need to consult such changes with someone from the documentation team. However we have no idea who could it be since Avital has left the project.